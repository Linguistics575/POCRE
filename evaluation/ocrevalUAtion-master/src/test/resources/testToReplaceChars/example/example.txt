ꝰ  ꝰ ū ꝙ   ꝑ  ſ ⁊ ſ ūoſſ⁊ ꝙ ōſōpōſſ ſ
ↄ -ūōſ ?rĳ.Scꙋāōſōāꝙßꝑ⁊ꝰꙋꝑſſſꝑ
