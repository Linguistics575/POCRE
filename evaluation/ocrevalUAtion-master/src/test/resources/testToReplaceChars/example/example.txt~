deꝰ nō pote illa facere. Sed potiꝰ
dicendū ꝙ creature non pt ia reciꝑe.vt videlicet
alid ſit viuū ⁊ mortuū ſimul ⁊ ſemel. Ita etiā im
plicat vt alis habeat liberū arbitriū ex quo poit
cauſe ſue inherere ⁊ ꝙ poit nō peccare.qr ſi nō po
te peccare.nō pōt cauſe ſue inherere cum pct ſit
ſpreto incōmutabili bono rebↄ mutabilibↄ inhere-
re. Spernere aūt vel nō ſpernere e ex litate arbi
?rĳ.Scꙋm etiā argutuz nō valet.qr ſi ga cōfir-
matiōis tere creature in naturā.ita ꝙ ex ncipĳs
eentialibↄ het nō poe peccare.tūc nō het ex ali-
quo dono acctali ⁊ ga nō poe deficere 7 peccare
ß haberet  ꝑ naturā.⁊ tūc eet deꝰ qꙋ e abſurdū
Hanc ſolutionē tangit ſctūs Tho.vbi ſupra in ſo
lutione vltimi arguti cū ꙋt ꝙ qcun alicui cre
ature Ɂuenit aliqꙋ accs qꙋ ex inﬂuentia ſuperio-
ris tmō ine.nō pōt inferior natura illud accñs
ꝑ ſe he niſi efficere nature ſuꝑioris ſicut ſi aer il-
lumīatur in actu ꝑ ignē nō pōt ee ꝙ ſit de natura
ſua lucidius īactu niſi fiat ignis.Dico ergo ꝙ cū
