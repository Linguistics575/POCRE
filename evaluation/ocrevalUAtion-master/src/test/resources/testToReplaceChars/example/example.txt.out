ꝰ ſt ꝰ ū ꝙ ñ ſt ꝑ q̇ ſ ⁊ ſ q̇ūoſſſſ⁊ ꝙ ſſōſſtōpōſſm̃ ſ
ↄ -ūōſſt ?rij.Scꙋām̃ōſr̄ōt́āꝙſſr̄ſſr̄ñßh̊ꝑ⁊ꝰꙋm̃ꝑſr̄ſt̃ſꝑ
